apps/web/lib/webhooks/subscriptions.tsx

```diff
@@ -2,10 +2,17 @@ import { WebhookTriggerEvents } from "@prisma/client";
 
 import prisma from "@lib/prisma";
 
-const getSubscribers = async (userId: number, triggerEvent: WebhookTriggerEvents) => {
+const getSubscribers = async (userId: number, eventTypeId: number, triggerEvent: WebhookTriggerEvents) => {
```

Comment:
Since this function is growing would it make more sense to use an options object?

```suggestion
const getSubscribers = async (options: GetSubscribersOptions) => {
```

Reply:
Yeah, good point. Will update 😄